docker-entrypoint-initdb.d/000_install_timescaledb.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
+# Checks to support bitnami image with same scripts so they stay in sync
+if [ ! -z "${BITNAMI_IMAGE_VERSION}" ]; then
+	if [ -z "${POSTGRES_USER}" ]; then
+		POSTGRES_USER=${POSTGRESQL_USERNAME}
+	fi
+
+	if [ -z "${POSTGRES_DB}" ]; then
+		POSTGRES_DB=${POSTGRESQL_DATABASE}
+	fi
+
+	if [ -z "${PGDATA}" ]; then
+		PGDATA="/opt/bitnami/postgresql/conf"
```

Comment:
Does `POSTGRESQL_DATA_DIR` not work here? https://github.com/bitnami/bitnami-docker-postgresql/blob/master/11/debian-9/rootfs/app-entrypoint.sh#L49